docs/_posts/2019-07-14-Magic-Mirror-Project-Introduction-and-Kick-Off-Notification.md

```diff
@@ -0,0 +1,110 @@
LM5lInLo7tqjqXjhLgTV7Vx2+DsHPjilepYX6Xi8hULOlzpvgbE0HBMWBVMPPbXJbPOU2mpQMBLSbyAp1iGmWaPue0XOMdWIbAStxKgoh99U+V/QzMksrOWEGK9/Wqlmgz+xxAMqR4VU9YIJVv35bQSG9Mzlfevd13A1qeQHI7NeUVj0aiphfVY0UbaV2YNrESTukDkQQYZ9qynr2R7lXgFF3RQRT8FNgVwepZ4r1/moJpz0XltxfStppA64tic/9zsqf3oLMY9ygL1UoFcL9zHXRK3sNRr32t4F0wKItX0MgyLkQsI5zxNoFP+SNzuOvlFjAhLTADmxjw9KD208FNPuss9gisOPq9lq5M3EHiiY4o0sFuTmoOOUZWx7IPBkv3WUf45zxA0nccjuC9kQgjJY6bVReCxWC+wzcnd65rpe398q1IDGU/eQCT1rhSsrsbI6ehm+RpS0u4ntOo5dpVLt9A3HWyw407SD29xWeHzf3O9j6a8GYHOoyzrndOT/q4exwr7jBC2k5XRmAwYbAOMYY6RU9R6W44xOf43fHThvpAqyoASNVc2oFG/VQTKHNIC4zOKgZY8/DyqFEtY3UNs5rxvAazixEQzOja2hH1x9q+cwK9xyXUA75Kg13b6S9//B+5d30USWAXrpyexvziBuLWrZuoVcGYoksrX96D5fYHH6BHyk1C2zha38Riv4UTsGlUsryM5AnXZKZ0B6ljO3RZgynVev4UiTTGVtKU+WceqfAYR/TqBj+ffcn6ofP3AwQeV9N79+/QZlyjicHSk6cPqQNH7xxz2/fGUFs4whBqZnYeOrSTdyZdxa3SxOQ19Er1q4owAWt6291rQ9UBXufO+u4e3CrBtKs+MTI4wvMBEtVdmMitdIQz/NJfd5CySk3dVK3Yn0iD3BaXTv1IiLvDgKgdKej2DjdX0X9rMLherenJ4MT4FYylstqaDv6X8DuqysE9Ng5bMNVz83g0wOl/R9uwgmPGht2fP6V7Q7XvKlw0xxTYjqjXBhOUySqQy7iK6luFSHeW8Yi8AbMZQgNgzJG8khRnwit5zrytz61nY1/9JdxcGn99kPhvgH0xlLOxGWYdeOAfS10Ukd++0hb1TA1Yr9S78eqVJ7NfACKna1SU6jlHNFLmdagCV9W9MT8TT14ZztIWpoz3F9HJcvgEL+V7f0fIzxluVbbpYCXkJ2rf++BDuFwUpHH70Dat3zvclywt8ZIJEqX2LW6S6MVdxiGLjLv98akb4d5Dls1bKnYZeFo+9rFL7MPafRvXIW3tByioo5jPpL2jH9GYGJlz3EGV1ij4NX0H8wWRQkJQxVsPG+y4WhytWyisVr+bssejaeNz/sX53+R3QVSakQlXiMck3mijM7nEv8A/8+LCH+BhfaW/SL8uUF6BTeJLUyqQC0kaaTKKDmomWOpxSH3yIn55fgG74AOF8nJ9nfDE75Iqs0AqaUIarXioCjKrIS2mqFc2EyXRJd/i7y0z7xoCR+qYafn6Or4r7O86vdhJnyNoHQtKVYADzvrnNvhdS7XNcpvHDtOOVoWYMGjQaFP1En/HQE+O/hl9FFjCdfjppsdrGu0D8HCkItJJzpeOeSTl6syZL+dptHueDEnGYNNIJ09QMk/ERerL8BaSjCRSSvJ5pLx3olTpCUxlC/49Mcwh/wDSvfdv/ynH3Ehej7W+XkynPfvhYqcNsZrXHdoN49gIhtErEQ/SOvEwoTIjMFBHGC5tcRw7NNjFsbsGSvjaxEWR84NsqTcUtcHwjXJFZq1d6/p8LKs7IS3C9bM5MnCdNJ1Yb3OCwvFy6p/iSHqLOuPqiBOU46MNvtWJPMSQZxxa1dLq5iz4LsP0TqSJ4RvQ1es9R5mb8VnaCrMPo72/j4spGCwOr8AHS/ywoFdaQVtEm58g7ZxH55506O/fvPNeuok1fTtH4iFyZDZvbe2DwbwNU9SefvP5f01zeDhRyrmJesJnDz5NY/3j6ZvHT7ECX0BiiMN0aHl00BrXZH75+LN07/YP0of3f8DpFipZyzPoAYwg1eW0iyJ60CF9+vgB11n2poUXi2nxxWb6H/+Hv0KCOhLrzeNHX9OGShLb0GH9Ii1xhO8VnLf+4/+S/vmf/x7XSWwAkA62o0KxgzeDrAbAfepbOOenHO95X6ZeTmeaMaytryKF1Sq+Nf36009xOD9NvTxta0eC2ZPm1+bop2xSkCS3047txH//o48xxtrIVv2qTbAx2EIv+HRQv674ZqWdvVI1hCc0el8ffkxhfte5aGUPydAAV2i6nvVzAujFAdeuXENyTJ9h19OD0W4vUlnLWlpahRFfJh8+VpGK9yKceYKu7Pvv6Yh/BPiEy7p7fvv///0uKMA85Fzk9PKuFu/fRfF1GLS1eCicyvNe/c13/uBa7XqjV4x3CWWN16OJXdOzxtcF14gsTHCtYCVgHqLkyOKcniXD8h1uCF0XmblmZ2bCRZIWhAccNfTjlFgH8fpza0UB3Z3l40dP0/jkGDpL/sZpCapC88swpTzcnJ5kEu9I9+7fY6JfTS9erqVdJtYuFK97kKA6AeiM2Oua1Pez0IwMkkGQyEeajbEX6UgTizYJQ0RMXhfzIICZy/vqyXemKwxKNGgs9LLIeeH0eMfycpkuHXmJiwUc+KdMStEhWBDMbymFUdCa23y8AI+8uMooByMkPsSXkJkj60O8HZ1dd8ZLxkNJaGBEY+QFN3CNuAwhasefKK8Arb7VWQq8qVvoJVLGSeAhCjJmWbpU8kpbP+riRjnCCVStT8ljPXJ80BGYJhGG9fadzzJOmT7WQUCZDjK8wXygP8JD0EyaWk/z5brn9vXZYB5pK27iLOzAuZDR70ibO3Fkqv6YLoxNpG8VAi7plWRYT8AGrr4W51xpsGpqJ8s3RF2b4i036GA7+553FGltoxLBNJWi+fa9n1I3k5WQ4zIdZDQim/DBKeqRG6Mkj++gw7n4IIXtEijnvtmcyUEeNK/TpGpTE5E51zsjHf
```

Comment:
Please move the inline image data to the separate image saved in `/asset/2019/` folder.

Reply:
Uploads are disabled.